docs/source/app_developers_guide/docker.rst

```diff
@@ -490,3 +461,234 @@ look similar to this:
 
   DATA: "b'\xa1fcCTdcH\x192B'"
   HEAD: "0c4364c6d5181282a1c7653038ec9515cb0530c6bfcb46f16e79b77cb524491676638339e8ff8e3cc57155c6d920e6a4d1f53947a31dc02908bcf68a91315ad5"
+
+
+
+Connecting To The REST API
+==========================
+
+From the Client Container
+-------------------------
+
+Use curl to confirm that you can connect to the REST API from the host.
+
+Enter the following command from the client container:
+
+.. code-block:: console
+
+  $ curl http://rest_api:8080/blocks
+
+
+From the Host Operating System
+------------------------------
+
+Use curl to confirm that you can connect to the REST API from the host.
+
+Enter the following command from a terminal window:
+
+.. code-block:: console
+
+  $ curl http://localhost:8080/blocks
+
+
+Container Overview
+==================
+
+The Client Container
+--------------------
+
+No Sawtooth components are automatically started in this container
+
+The client container's primary uses:
+
+* Submitting transactions
+* Running sawtooth CLI commands
+
+Log into this container by running this command from the host computer's terminal:
+
+.. code-block:: console
+
+  % docker exec -it sawtooth-client-default bash
+
+
+The Validator Container
+-----------------------
+
+* Runs a single validator
+* Available to the other containers and host on TCP port 4004
+* Hostname: validator
+
+Log into this container by running this command from the host computer's
+terminal:
+
+.. code-block:: console
+
+  $ docker exec -it sawtooth-validator-default bash
+
+To see which components are running, run this command from the container:
+
+.. code-block:: console
+
+  $ ps -ef | grep python | grep -v grep
```

Comment:
With docker, the command you give as the entrypoint for an image will be started with PID 1 within the container and PPID 0. A better command for this would be something like `ps --pid 1 fw`. That will just show whatever the entrypoint for the container was, which should be what they want to see.